var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -41,6 +41,9 @@ class Ncurses(AutotoolsPackage):
     variant('symlinks', default=False,
             description='Enables symlinks. Needed on AFS filesystem.')
 
+    variant('wide', default=True,
+            description='Enable to build ncuresesw, disable to build ncurses')
```

Comment:
I asked about conflicting variants in #3849 and the consensus seemed that a variant was the best way to do it.